extensions/amp-story/0.1/amp-story-system-layer.css

```diff
@@ -136,7 +136,17 @@
   background-position-y: calc(50% - 2px) !important;
 }
 
-.i-amphtml-story-no-sharing .i-amphtml-story-share-control {
+.i-amphtml-story-info-control {
+  background-image: url('data:image/svg+xml;charset=utf-8,<svg xmlns="http://www.w3.org/2000/svg" width="24px" height="24px" viewBox="0 0 24 24" fill="#FFFFFF"><path d="M0 0h24v24H0z" fill="none"/><path d="M12 2C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm1 15h-2v-6h2v6zm0-8h-2V7h2v2z"/></svg>') !important;
+  display: none !important;
+}
+
+.i-amphtml-embedded .i-amphtml-story-info-control {
```

Comment:
I may be missing something here, but it looks like this wouldn't work because the ``i-amphtml-embedded`` class isn't set in the shadow DOM?

Apparently, all we'd have to do would be to add this class ourselves when building the system layer. Once we retrieve the ``viewer`` through ``viewerForDoc``, there's this [``viewer.isEmbedded``](https://github.com/ampproject/amphtml/blob/master/src/service/viewer-impl.js#L519) method.
Also, I think it'd be great to add this test in the ``amp-story`` file, so we don't even build the component if we don't need to

Reply:
Done